stories/Loaders/Loaders.story.jsx

```diff
@@ -16,10 +16,10 @@ const descriptions = {
   circularLoader: `CircularLoaders are used to give the user the impression of loading while a component is not ready to be displayed.`,
 };
 
-const progressBarExamplesTab = (
+const renderExampleTab = examples => (
```

Comment:
how about change the examples var to `componentExample` to make it clearer ?
WDYT?